playwright-tests/tests/upgrade/upgradePmm.spec.ts

```diff
@@ -0,0 +1,54 @@
+import {expect, test} from '@playwright/test';
+import {api} from '@api/api';
+import apiHelper from "@api/helpers/apiHelper";
+import grafanaHelper from '@helpers/GrafanaHelper';
+import HomeDashboard from '@pages/HomeDashboard.page';
+import {SettingProperty} from "@api/settingsApiV1";
+
+test.describe('General Upgrade PMM tests', async () => {
+  test.describe.configure({ retries: 0 });
+    let pmmVersion: number;
+
+  test.beforeAll(async () => {
+    if (!pmmVersion) {
+      pmmVersion = (await api.serverV1.getPmmVersion()).minor;
+    }
+  });
+
+  test.beforeEach(async ({ page }) => {
+    await grafanaHelper.authorize(page);
+    await apiHelper.confirmTour(page);
+    await page.goto('');
+  });
+
+  test('PMM-T288 Verify user can see Update widget before upgrade [critical] @pre-upgrade @ovf-upgrade @ami-upgrade @pmm-upgrade', async ({
+    page,
+  }) => {
+    const homeDashboard = new HomeDashboard(page);
+    await homeDashboard.pmmUpgrade.verifyUpgradeWidget();
+  });
+
+  test('PMM-T3 Verify user is able to Upgrade PMM version [blocker] @pmm-upgrade @ovf-upgrade @ami-upgrade', async ({ page }) => {
+    const homeDashboard = new HomeDashboard(page);
+
+    await homeDashboard.upgradePMM();
+  });
+
+  if (pmmVersion < 36) {
```

Comment:
Move this as we talked about to its folder and spec file PMM-T1659 both tests

Reply:
Which one feature do you think it is? Asking as I don't think this is a dedicated feature.

Reply:
moved to "configuration" feature